WPMedia/engine-theme-sdk
# Updated twittericon

## Description

Update with new Twittericon

## Jira Ticket

- [THEMES-1440](https://arcpublishing.atlassian.net/browse/THEMES-1440)

## Acceptance Criteria

https://www.figma.com/file/NBjMF1v4mzwv9jkn7NEq9V/Themes-Block-Styling-2.0?type=design&node-id=10811%3A147806&mode=design&t=jG5oCFMUMr3b84Ro-1

## Test Steps

1. Checkout branch - git checkout THEMES-1440
2. Update dependencies - npm i
3. Run footer-block from pagebuilder 
4. Check out the Twitter icon in footer-block


## Review Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).



[THEMES-1440]: https://arcpublishing.atlassian.net/browse/THEMES-1440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
